mini_pupper_gazebo/README.md

```diff
@@ -2,10 +2,12 @@
 
 You can also play with Mini Pupper with only your laptop or PC.  
 **Raspberry Pi does not have enough resources to simulate.**
+**Notice: Setting the param `/use_sim_time` as `true` is required for simulation.**
 
 ![nav](../imgs/instruction.gif)
 
-Launch the Gazebo simulator with the following command.
+Launch the Gazebo simulator with the following command.  
+The param `/use_sim_time` is set to `true` by this launch file.
```

Comment:
`/use_sim_time`
or `use_sim_time`?

Reply:
/use_sim_time is accurate. It will automatically be /use_sim_time if we declare without a namespace, but to avoid misunderstandings, I'll use the correct notation.